playground/src/app.critical.scss

```diff
@@ -207,6 +256,39 @@
       }
     }
   }
+
+  &--light {
+    .panel-header {
+
+      * {
```

Comment:
I'd love if we could reduce the usage of `*` in the overall playground codebase 😅  Doesn't simple inheritance work?  Especially cause it is just a font-color?